Demo/Application/Base/BraintreeDemoContainmentViewController.m

```diff
@@ -74,9 +74,11 @@ - (void)setLatestTokenizedPaymentString:(NSString*)latestNonceString {
 }
 
 - (void)updateStatus:(NSString *)status {
-    [(UIButton *)self.statusItem.customView setTitle:NSLocalizedString(status, nil) forState:UIControlStateNormal];
-    [(UIButton *)self.statusItem.customView setTitleColor:UIColor.blackColor forState:UIControlStateNormal];
-    NSLog(@"%@", ((UIButton *)self.statusItem.customView).titleLabel.text);
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
We were not handling the status update for any of our views on the main thread - this now prevents the warnings we were seeing in Xcode.